joomla/joomla-cms
# [Sql azure] Normalize joomla install sql file

### Summary of Changes

Normalizes all sql azure joomla install file.
- always use `SET IDENTITY_INSERT [#__xxxx] ON/OFF;`
- `INSERT` to `INSERT INTO`
- Quote all table names and db field names
- Always use `'` to quote string values.
- Code style
### Testing Instructions

Mainly code review. (easier to check code diff without white spaces changes) https://github.com/joomla/joomla-cms/pull/11725/files?w=1
If anyone could test an install 3.6.x would be great. Don't have sql azure to test,
### Documentation Changes Required

None.
